exa/core/data.py

```diff
@@ -169,6 +169,12 @@ def from_tarball(cls, yml=None, qet=None):
             df = pd.read_parquet(qet)
         return cls(df=df, **yml)
 
+    def read_csv(self, *args, **kws):
+        """Convenience method for pandas read_csv"""
+        self.source = pd.read_csv
+        self._data = self.source(*args, **kws)
```

Comment:
Thanks for the contribution. I think the intent here is to create an alternate constructor e.g.
```
from exa import Data
data = Data.read_csv("~/my.csv")
```
This can be accomplished by making `read_csv` a classmethod:
```
@classmethod
def read_csv(cls, *args, **kws):
    return cls(df=pd.read_csv(*args, **kws)
```